doc/modules/mod_event_pusher_rabbit.md

```diff
@@ -10,10 +10,10 @@ receiver) along with the message body.
 * **group message sent/received** - Carries the user id and the room id
 (full jids by default) along with the message body.
 
-All these notifications are sent as JSON strings to RabbitMQ topic exchanges.
-Each type of the notifications is sent to it's dedicated exchange. There are
-three exchanges created on startup of the module, for presences, private
-messages and group chat messages related events.
+All these notifications are sent as JSON strings to RabbitMQ exchanges. Type
+of exchanges can be chosen as desired. Each type of the notifications is sent
+to it's dedicated exchange. There are three exchanges created on startup of the
```

Comment:
Now I think if the exchange should be deleted on node shutdown. What if one node in a cluster goes down? I think they should not be deleted. You could have an API to delete them explicitly wired to mongooseimctl if needed.

Reply:
Good spot. Maybe mongooseimctl API is to much for the first iteration? What should we do with that for now?

Reply:
Maybe we can check if we're the last node in the cluster?

Reply:
I think Mongoose should just check if the appropriate toplogy is in place, that's all. So that if you take down a cluster for Maintance RabbitMQ still retains it's messages etc. Then if you start the cluster up again all is configured. 


But I'd go for an API to delete the topology anyway.

Reply:
You mean `mongooseimctl` API for that?

Reply:
Not necessarily.

Reply:
I'll deal with that in the separate PR.